src/objects.js

```diff
@@ -1,41 +1,65 @@
 const createPerson = (name, age) => {
-  // your code here
+  return {
+    name: name,
+    age: age
+  }
```

Comment:
if the key of the object matches the name of the constant you are assigning it you can just write the constant, i.e.
```
return {
  name,
  age,
}
```